source/main/physics/Beam.cpp

```diff
@@ -5932,32 +5923,28 @@ bool Beam::LoadTruck(
 							dash->loadDashBoard("default_dashboard7000_mph.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard7000_mph.layout", true);
-						}
-						else
+						} else
 						{
 							dash->loadDashBoard("default_dashboard3500_mph.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard3500_mph.layout", true);
 						}
-					}
-					else
+					} else
 					{
 						if (engine->getMaxRPM() > 3500)
 						{
 							//7000 rpm tachometer thanks to klink
 							dash->loadDashBoard("default_dashboard7000.layout", false);
 							// TODO: load texture dashboard by default as well
 							dash->loadDashBoard("default_dashboard7000.layout", true);
-						}
-						else
+						} else

```

Comment:
This style of writing `else` is undesirable. I know it's everywhere and that I introduce inconsistencies, but it's just pain to look at for me. I'd like to eliminate it completely.